examples/compose/plot_transformed_target.py

```diff
@@ -73,91 +69,93 @@
 # prediction. Subsequently, a logarithmic function is used to linearize the
 # targets, allowing better prediction even with a similar linear model as
 # reported by the median absolute error (MAE).
+from sklearn.metrics import mean_absolute_error, r2_score
+
+
+def compute_score(y_true, y_pred):
+    return {
+        "R2 score": f"{r2_score(y_true, y_pred):.3f}",
+        "MAE": f"{mean_absolute_error(y_true, y_pred):.3f}",
+    }
```

Comment:
The example was using the median_absolute_error. Is it intended to use the mean_absolute_error ? if so, references to median_absolute_error should be replaced. On a side note, it looks like MAE usually means mean_abs_error while median_abs_error is usually abreviated MedAE.